.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -9,14 +9,20 @@ assignees: ""
 ### Release checklist for GitHub contributors
 
 - [ ] All PRs/issues attached to the release are merged.
-- [ ] All the badges on the README are passing.
-- [ ] License information is verified as correct. If you are unsure, please comment below.
-- [ ] Locally rendered documentation contains all appropriate pages, including API references (check no modules are
+- [x] All the badges on the README are passing.
+- [x] License information is verified as correct. If you are unsure, please comment below.
+- [x] Locally rendered documentation contains all appropriate pages, including API references (check no modules are
   missing), tutorials, and other human written text is up-to-date with any changes in the code.
-- [ ] Installation instructions in the README, documentation and on the website (e.g., diffpy.org) are updated and
-  tested
-- [ ] Successfully run any tutorial examples or do functional testing in some other way.
-- [ ] Grammar and writing quality have been checked (no typos).
+- [x] Installation instructions in the README, documentation and on the website (e.g., diffpy.org) updated.
+- [x] Successfully run any tutorial examples or do functional testing with the latest Python version
+- [x] Grammar and writing quality have been checked (no typos).
```

Comment:
@cadenmyers13 Seems like these are all checked by default! 

This file is used as a template when the user clicks "New issue"